doc/netplan.md

```diff
@@ -624,6 +624,25 @@ wpasupplicant installed if you let the ``networkd`` renderer handle wifi.
           and ``adhoc`` (peer to peer networks without a central access point).
           ``ap`` is only supported with NetworkManager.
 
+     ``bssid`` (scalar)
+     :    If specified, directs the device to only associate with the given
+          access point.
+
+     ``band`` (scalar)
+     :    Possible bands are ``a`` (for 5GHz 802.11a) and ``bg``
+          (for 2.4GHz 802.11), do not restrict the 802.11 frequency band of the
+          network if unset (the default).
+
+     ``channel`` (scalar)
+     :    Wireless channel to use for the Wi-Fi connection. Because channel
+          numbers overlap between bands, this property takes effect only if
+          the ``band`` property is set, too.
+
+     ``seen-bssids`` (sequence of scalars)
```

Comment:
I'd like to ask Steve for an opinion. So I wanted to include this key because of the NM read-write plugin, so that when the seen-bssids state changes, users can see that also in the netplan configuration and not only via NM. Of course that was the idea, but I'd like to hear from Steve if maybe it's not really needed?
Anyway, for now the one thing I'd recommend either way is to not include it in the documentation, or explicitly mentioning that it's a read-only variable that is used only to track NM state when using the read-write plugin.

Reply:
Done. Stated it explicitly. If we do not include it, our test suite will an error about missing/non-documented fields.